packages/framework/src/Resources/config/services.yaml

```diff
@@ -4,6 +4,7 @@ imports:
 parameters:
     # Use Shopsys validator factory for fp/jsformvalidator-bundle
     fp_js_form_validator.factory.class: Shopsys\FrameworkBundle\Form\JsFormValidatorFactory
+
```

Comment:
I am not a fan of having unnecessary spaces, can it be set to not include them?

Reply:
nope, at this moment yaml standards for spaces accept only 1 parameter: what max level do you want to have space between group/row...you can't apply exceptions, it's only for everything

if you want, I can implement exceptions to library but if I do it and I will wait next 3 years I don't see any sense in doing it, sorry